pyproject.toml

```
@@ -28,6 +28,7 @@ test = [
     "flake8-bugbear",
     "black",
     "scikit-learn",
+    "tensorflow >=2.5.0",
 ]
 dev = ["pre-commit"]
 
```

Seconding Chris' concern, I think adding a new category for the samples is fine. The used function seems to be already available in 2.0.
```suggestion
samples = ["tensorflow >=2.0.0"]
```

---

Restricting the version number was mainly due to the automatic tests, because pip was backtracking while installing the dependencies and took hours. But using 2.0.0 should probably also be fine.